gitlab-runner-17.8.yaml

```diff
@@ -135,8 +135,6 @@ subpackages:
 
 update:
   enabled: true
-  # Requires manual steps when updating
-  manual: true
```

Comment:
I think these lines should be removed in your follow-up PR: until that lands, I think we will still need to manually verify any new versions that come in.

Reply:
I have added the check for this, I will also include this check for all the other package when I will add the entrypoint subpackages

Reply:
I think we can remove the manual:true with this check